### ISSUE TITLE
Make logger object available in the LogsterParser class

### ISSUE THREAD

**Issue by bigunyak** [etsy/logster#95]
Hi there,

Not a bug but a feature request. :)
Would be convenient to have logger object available in parser classes for debugging purposes.

**Comment by benburry** [comment]
Absolutely. Would you feel up to submitting a pull request with the change?
